ngraph/core/src/op/interpolate.cpp

```diff
@@ -239,11 +239,20 @@ void op::v4::Interpolate::validate_and_infer_types()
         return;
     }
 
```

Comment:
Please add type checks for other inputs.

Reply:
Done.

Reply:
Please double-check it.

Reply:
My bad. I found:)